server/services/user.service.js

```diff
@@ -552,7 +568,13 @@ export const updateVendor = async ({ updatedVendor, user }) => {
     Array.prototype.push.apply(updatedVendor.vendor.socialMedia, user.vendor.socialMedia);
   }
 
-  const vendor = { ...user.vendor, ...updatedVendor.vendor };
+  const vendor = {
+    ...user.vendor,
+    ...updatedVendor.vendor,
+    verified: updatedVendor.vendor
```

Comment:
Also here, you didn't that non-vendor attributes can be sensitive too 😏 

```suggestion
    verified: containsSensitiveInfo(updatedVendor) ? false : user.vendor.verified,
```